package.json

```diff
@@ -51,7 +51,7 @@
     "less": "2.6.0",
     "less-plugin-autoprefix": "1.5.1",
     "nomnom": "1.8.1",
-    "openlayers": "openlayers/ol3#82f6fb9",
+    "openlayers": "openlayers/ol3#d5fb6dd",
```

Comment:
We cannot use v3.14.1 because we need this commit: https://github.com/fredj/ol3/commit/916127d8f69eb163bf354ad8124a7797cbbdae98